src/tracks/playabletrack/wavetrack/ui/WaveClipAdjustBorderHandle.cpp

```diff
@@ -235,13 +256,13 @@ class AdjustClipBorder final : public WaveClipAdjustBorderHandle::AdjustPolicy
       const auto eventT = viewInfo.PositionToTime(viewInfo.TimeToPosition(mInitialBorderPosition, event.rect.x) + dx, event.rect.x);
 
       const auto offset = sampleCount(floor((eventT - mInitialBorderPosition) * mClips[0]->GetRate())).as_double() / mClips[0]->GetRate();
-      const auto t = std::clamp(mInitialBorderPosition + offset, mRange.first, mRange.second);
+      const auto t = mInitialBorderPosition + offset;
```

Comment:
Is the clip stretching handle completely correct, or must you also restore the `std::clamp`?  What happens below if snapping is disabled?  I don't see where `t` is limited by `mRange` on that path.

Reply:
Thank you: that's a change I forgot to revert.